src/OneExplorer/ConfigObject.ts

```diff
@@ -39,7 +41,7 @@ interface Artifact {
   /**
    * A full path in file system
    */
-  path?: string
+  path: string;
```

Comment:
@jyoungyun Applied :-D

Reply:
Before, `Artifact` was used for query too, with an undefined `path`.
Here, I have replaced that with `ArtifactAttr: {type, ext}` and used `Artifact` with path only.